Gemfile

```diff
@@ -12,10 +12,7 @@ gem 'coffee-rails', '~> 5.0'
 gem 'date_validator'
 gem 'faraday', '~> 1.10.3'
 gem 'net-http' # needed to undo a conflict with system libs
-gem 'govuk_notify_rails'
-# we need the extra is_csv parameter available in 5.2 and above
-# TODO: update code to work with latest version
-gem 'notifications-ruby-client', '>= 5.2', '< 6.0.0'
```

Comment:
I assume the removal of this gem is intentional and no-longer needed with the newer notify gem?

Reply:
Yes, this gem is a is a transitive dependency of the other gem, thus no need to declare it here. We only had it here temporarily because we needed a specific version until we made the code changes in this PR 👍